### DISCUSSION TITLE
The schedule for releasing the "Saving the grid's sorting state as part of GridSetting" feature

### DISCUSSION THREAD

**Discussion by Kerminator1973** [vikramlearning/blazorbootstrap#680] [General]
Do you have any schedule for the "Saving the grid's sort state as part of GridSetting" feature?

There is a comment "_This functionality will be included in future releases_" on the official website. However, we'd like to use this feature within a few months. Is it realistic that this feature will be released before fall 2024?

Could you please tell us about the complexity of this feature, I mean, is it difficult to code this feature for an ordinary developer who hasn't explored the source code of the project yet?

**Comment by gvreddy04** [comment]
@Kerminator1973 Thank you for reaching us on this. I'll share the details by next week.

**Reply by Kerminator1973** [reply]
Thank you very much! I created a repository that contains a minimal functionality application to illustrate the expected implementation of the GridSettingsChanged event and appended some additional information.

**Reply by gvreddy04** [reply]
@Kerminator1973 Pls share the repository link.

**Reply by Kerminator1973** [reply]
I used the debugger to analyze the source code of the Grid component and couldn't find any way to add a reference to items to the GridSettingsChanged event. Unfortunately, any code change will break the current API. Could you please look at my conclusions at https://github.com/Kerminator1973/bbTableCallback, I hope I missed some useful way to do this.

**Comment by Kerminator1973** [comment]
I am so sorry! I was a little bit tired last night and forgot to share it: https://github.com/Kerminator1973/bbTableCallback
